tables/v1/aws_roles.sql

```diff
@@ -1,19 +1,20 @@
 SET SEARCH_PATH TO v1;
 
 CREATE TABLE IF NOT EXISTS aws_roles (
+    id            SERIAL                    NOT NULL  PRIMARY KEY,
     role_arn      TEXT                      NOT NULL,
     environment   TEXT                      NOT NULL,
     namespace_id  INTEGER                   NOT NULL,
     created_at    TIMESTAMP WITH TIME ZONE  NOT NULL  DEFAULT CURRENT_TIMESTAMP,
     created_by    TEXT                      NOT NULL,
-    PRIMARY KEY (role_arn, environment, namespace_id),
     UNIQUE (environment, namespace_id),
```

Comment:
Might was well add the `role_arn` into the unique index as well.
```suggestion
    UNIQUE (role_arn, environment, namespace_id),
```

Reply:
Then we could have multiple `role_arn`s for the _same_ `environment` / `namespace` combo, which is invalid.

Reply:
> Then we could have multiple role_arns for the same environment / namespace combo, which is invalid.

🤦